BlazarUI/app/stylus/page/settings.styl

```diff
@@ -170,10 +175,22 @@
     font-size 48px
     color #cbd6e2
     padding-bottom 20px
-  
+
   & p
     margin-top 5px
     width 100%
     font-family HelveticaNeue
     font-size 20px
     color #cbd6e2
+
+.notifications__channelList
```

Comment:
Small nit for these class names - don't use camelCase, just separate with dashes: `.notifications__channel-list`

We're using [BEM](http://getbem.com/introduction/) to organize our css